### ISSUE TITLE
App_Start code not created

### ISSUE THREAD

**Issue by grosch** [ninject/Ninject.Web.Common#37]
Using Visual Studio 2017 I installed Ninject.Web.Common.WebHost via NuGet.  It updated the pakcages.config but it did not create Any code in the App_Start directory for the kernel.

**Comment by scott-xu** [comment]
You have to create the file manually.

**Comment by scott-xu** [comment]
Source and configuration file transformations are not applied when a package is installed in a project using Package References in project files.
see https://docs.microsoft.com/en-us/nuget/create-packages/source-and-config-file-transformations

**Comment by scott-xu** [comment]
Try 3.3.1 please

**Comment by grosch** [comment]
I ran the below command.  Still didn't create any file in App_Start.  

I'm using VS Professional 2017 version 15.6.6

> PM> Install-Package Ninject.Web.Common -Version 3.3.1
Attempting to gather dependency information for package 'Ninject.Web.Common.3.3.1' with respect to project 'LabSORService', targeting '.NETFramework,Version=v4.6.1'
Gathering dependency information took 657.26 ms
Attempting to resolve dependencies for package 'Ninject.Web.Common.3.3.1' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'Ninject.Web.Common.3.3.1'
Resolved actions to install package 'Ninject.Web.Common.3.3.1'
Found package 'Ninject 3.3.3' in 'C:\Users\grosch\Documents\LabSOR\LabSORService\packages'.
Retrieving package 'Ninject.Web.Common 3.3.1' from 'nuget.org'.
  GET https://api.nuget.org/v3-flatcontainer/ninject.web.common/3.3.1/ninject.web.common.3.3.1.nupkg
  OK https://api.nuget.org/v3-flatcontainer/ninject.web.common/3.3.1/ninject.web.common.3.3.1.nupkg 102ms
Installing Ninject.Web.Common 3.3.1.
Package 'Ninject.3.3.3' already exists in folder 'C:\Users\grosch\Documents\LabSOR\LabSORService\packages'
Added package 'Ninject.3.3.3' to 'packages.config'
Successfully installed 'Ninject 3.3.3' to LabSORService
Adding package 'Ninject.Web.Common.3.3.1' to folder 'C:\Users\grosch\Documents\LabSOR\LabSORService\packages'
Added package 'Ninject.Web.Common.3.3.1' to folder 'C:\Users\grosch\Documents\LabSOR\LabSORService\packages'
Added package 'Ninject.Web.Common.3.3.1' to 'packages.config'
Successfully installed 'Ninject.Web.Common 3.3.1' to LabSORService
Executing nuget actions took 3.52 sec
Time Elapsed: 00:00:04.2522510

**Comment by scott-xu** [comment]
Run this please:

 >Install-Package Ninject.Web.Common.**WebHost** -Version 3.3.1

**Comment by scott-xu** [comment]
I checked it just now. It works fine. Could you please send a copy of you application to me?

**Comment by scott-xu** [comment]
BTW, you can setup binding redirection in your config file under runtime section. For example:
``` xml
  <runtime>
    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
      <dependentAssembly>
        <assemblyIdentity name="Ninject.Web.Common" publicKeyToken="c7192dc5380945e7"/>
        <bindingRedirect oldVersion="0.0.0.0-3.2.1.0" newVersion="3.2.1.0"/>
      </dependentAssembly>
    </assemblyBinding>
  </runtime>
```

**Comment by grosch** [comment]
I really wish I could send you the project but unfortunately it's pretty large and the company considers it IP at this point :(  I created a brand new solution and installed ninject and it worked without issue, other than having to add "using Ninject.Web.Common.WebHost;" to the generated file.  So, at this point, I think I'm just not going to be able to use ninject for this project.  The people I inherited it from did "something" that's making it not play nice.
